### ISSUE TITLE
Assumption about theta domain in calculating growth rate

### ISSUE THREAD

**Issue by d7919** [bpatel2107/pyrokinetics#70]
When loading eigenvalues in the super GKCode method the field amplitude is integrated over theta and then divided by 2pi. Presumably this should actually by divided by the extent of the parallel grid so that this is an average.

https://github.com/pyro-kinetics/pyrokinetics/blob/6716b3bdbb67294d6907ae2283b0c29e38678220/pyrokinetics/gk_code/GKCode.py#L128

This should be fine for CGYRO which (I think) always has a two pi extent, but wouldn't be correct for GS2 in general.

**Comment by bpatel2107** [comment]
For linear runs all codes are converted to ballooning space so one `kx` and the full ballooning angle range.  We can definitely change this, though because we differentiate a logarithm it doesn't technically matter and won't change the growth rate

`
ln(2A) - ln(2B) = ln(A) - ln(B)
`

This isn't used in the mode frequency. 

Could change it to

```python
square_fields.sum(dim="field").integrate(coord="theta") / (max(square_fields.theta) - min(square_fields.theta))
```

Or remove it entirely. Thoughts?

**Comment by d7919** [comment]
If it's not needed then I'd just remove it.

**Comment by bpatel2107** [comment]
Fixed in 6c59544
